### ISSUE TITLE
The "self-update" command is broken

### ISSUE THREAD

**Issue by aik099** [scrutinizer-ci/ocular#25]
Steps to reproduce:

1. download latest phar from https://scrutinizer-ci.com/ocular.phar
2. show it's version using `php ocular.phar --version`:

```
ocular version 8e0a8c7f085bc4857bd52132833679dcfd504fc1
```

3. run `self-update` command using `php ocular.phar self-update`:

```
#!/usr/bin/env php
Updating from 8e0a8c to #!/usr.
```

What not right:

1. the `#!/usr/bin/env php` is shown (something wrong with phar creation process probably)
2. update even from latest version possible (new version check is broken)
3. the `#!/usr` is shown instead of new version (new version check is broken)

**Comment by tremani** [comment]
Yes. Also, https://scrutinizer-ci.com/ocular.phar is two years outdated, keeping that one up-to-date would be a quick workaround for most users.

**Comment by Majkl578** [comment]
@schmittjoh This can be closed too now since the command was removed.
